README.md

```diff
@@ -78,7 +78,7 @@ Also, check the [example](example) directory.
 | validate           | `boolean`                                                                           | optional | When `true`, attempts source map validation before upload if rewriting is not enabled. It will spot a variety of issues with source maps and cancel the upload if any are found. Defaults to `false` to prevent false positives canceling upload.                                                                                                                               |
 | stripPrefix        | `array`                                                                             | optional | When paired with `rewrite`, will remove a prefix from filename references inside of sourcemaps. Useful for removing a path that is build-machine-specific. Note that this will NOT change the names of uploaded files.                                                                                                                                                          |
 | stripCommonPrefix  | `boolean`                                                                           | optional | When paired with `rewrite`, will add `~` to the `stripPrefix` array. Defaults to `false`.                                                                                                                                                                                                                                                                                       |
-| sourceMapReference | `boolean`                                                                           | optional | Prevents the automatic detection of sourcemap references. Defaults to `false`.                                                                                                                                                                                                                                                                                                  |
+| sourceMapReference | `boolean`                                                                           | optional | Determines whether CLI should automatically emit sourcemap references. By default it will store a `Sourcemap` header with minified files so that sourcemaps are located automatically, without the need for `sourceMappingURL` if the tool can detect a link. Defaults to `true`.                                                                                               |
```

Comment:
```suggestion
| sourceMapReference | `boolean`                                                                           | optional | Determines whether sentry-cli should attempt to link minified files with their corresponding maps. By default, it will match files and maps based on name, and add a `Sourcemap` header to each minified file for which it finds a map. Can be disabled if all minified files contain `sourceMappingURL`. Defaults to `true`.                                                                                           |
```